### ISSUE TITLE
Crash in the development enviroment "Cannot get config value before config is loaded"

### ISSUE THREAD

**Issue by CKenJa** [Engine-Room/Flywheel#210]
### Describe the Bug

I updated build.gradle in the development environment to use Create0.5.1f and it crashes when I runClient.

### Reproduction Steps

1. Build a forge mod development environment on IntteliJIdea
2. Add dependencies on Create and Flywheel
3. Reload Gradle and genInttelliJRuns
4. debug runClient

### Expected Result

Minecraft launches without crashing.

### Screenshots and Videos

[crash-2023-11-03_18.57.19-client.txt](https://github.com/Jozufozu/Flywheel/files/13248935/crash-2023-11-03_18.57.19-client.txt)

### Operating System

Windows 11

### What is your GPU?

Nvidia Geforce GTX 3060

### Mod Version

0.6.10

### Minecraft Version

1.19.2

### Loader Version

43.2.4

### Other Mods

Create, JEI, curios, Relocate(my developping mod)

**Comment by FortressNebula** [comment]
+1

Also happens on 1.20.1, with Forge 47.2.18, with Create, JEI, Farmer's Delight, Create: Deco, and Create: Alloyed (which is developing mod)

**Comment by Rabbitminers** [comment]
> +1
> 
> Also happens on 1.20.1, with Forge 47.2.18, with Create, JEI, Farmer's Delight, Create: Deco, and Create: Alloyed (which is developing mod)

Did you find a solution? I'm having the same issue in an [architectury project](https://github.com/Rabbitminers/Extended-Bogeys) on the forge side (47.1.44) with Flywheel 0.6.9-5.

**Comment by slava110** [comment]
Same problem. Minecraft `1.20.1`, Forge `47.2.0`, Flywheel `0.6.10-8`

**Comment by denkisikinoneko** [comment]
Same issue, Minecraft `1.20.1`, Forge `47.2.30`, Flywheel `0.6.10-7`

**Comment by slava110** [comment]
This error is caused by problems in other mods. In my case, I wasn't registering Create movement behaviour right. In case you have this error - try searching for problems in your mod or other mods

**Comment by zyxkad** [comment]
same 
[crash-2024-05-24_09.35.09-client.txt](https://github.com/Engine-Room/Flywheel/files/15436933/crash-2024-05-24_09.35.09-client.txt)

**Comment by Ironnoob73** [comment]
same issue, minecraft 1.20.1, forge 47.3.0, flywheel 0.6.10-7
All mavens use "implementation" will resolve it.

**Comment by hjake123** [comment]
Same issue. How are people figuring out where the issue is?

**Comment by EpicVon2468** [comment]
Also having this issue

**Comment by BetaFoprhoton** [comment]
I have been troubled by this problem for several months now, it's so confusing that it seems to be caused by different problems and the source of the problem seems to be unknown. Hope someone could figure it out.

**Comment by zyxkad** [comment]
Basically this error is when some part of your mod throws some error, and if flywheel is not initialized, Minecraft will ignore your error but only throws flywheel's error.

Three solutions for this:

1. place try - catch on every part of your code that may cause errors. And print the error manually instead of throw it to top handler.
2. remove flywheel and other mods that will do some random logics when error occures.
3. test in production envionment.

**Comment by EpicVon2468** [comment]
@zyxkad Yeah, after a while I figured out it was _my_ config that wasn't registered properly.
It's still weird that it would print this as a flywheel issue into the logs without ever mentioning the real cause though.

**Comment by BetaFoprhoton** [comment]
By the way, my issue has solved as well. I used kotlin to write my project but I forgot to add kff in my dependencies.

**Comment by IThundxr** [comment]
This is caused by a error being thrown very early during mod loading, which results in forge blaming flywheel
